examples/demo/package.json

```diff
@@ -9,6 +9,9 @@
   "dependencies": {
     "@nuxtjs/pwa": "^3.0.0-beta.20",
     "cross-env": "^7.0.2",
+    "elliptic": "6.5.3",
```

Comment:
Sorry just reviewing the dependencies, but perhaps we can remove this since it's not being used, wdyt?